plugin/pulpcore/plugin/download/http.py

```diff
@@ -44,6 +44,13 @@ class HttpDownloader(BaseDownloader):
     then closed when the download is complete. A session that is passed in will not be closed when
     the download is complete.
 
+    If a session is not provided, the one created by HttpDownloader uses non-default timing values.
+    Specifically, the "total" timeout is set to None and the "sock_connect" and "sock_read" are both
+    5 minutes. For more info on these settings, see the aiohttp docs:
+    http://aiohttp.readthedocs.io/en/stable/client.html#ssl-control-for-tcp-sockets
```

Comment:
Same link here.

Reply:
yeah these links aren't right. I'm pushing fixes in my next commit